missilewars-plugin/src/main/java/de/butzlabben/missilewars/commands/MWCommands.java

```diff
@@ -98,24 +98,24 @@ public void pasteCommand(CommandSender sender, String[] args) {
         Player player = (Player) sender;
 
         if (args.length < 1) {
-            player.sendMessage(Messages.getPrefix() + "§cMissile needed.");
+            player.sendMessage(Messages.getMessage("command.missile_needed"));
```

Comment:
instead of relying on magic values, we should rather use enums, to safely identify and retrieve values.
It could look something like this:
```
@Getter
enum Message {
  COMMAND_MISSILE_NEEDED("command.missile_needed"), ...

  private final String path;

  public Message(String path) {
    this.path = path;
  }
}
```

Then you could have extra helper functions only working with the enum. This will prevent bugs in the future and helps us keeping a close eye and all messages and possible translations.

Reply:
First, in this PR I only wanted to make the messages configurable and not refactor them. Secondly, we did not use an enum in the [Config.java](https://github.com/RedstoneFuture/missilewars/blob/a2ff87644e7391db045ef5a34cba336548838cc1/missilewars-plugin/src/main/java/de/butzlabben/missilewars/Config.java#L180-L182) either. If, then one would have to adapt that there also. Why an enum?

Reply:
Thanks for implementing the enum, although I would have been fine with addressing it later